### PR TITLE
Drop `tif`, `tiff`, `webp` and `bmp` from supported images.

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: spec/lib/upload_creator_spec.rb [hunk 1]
```diff
@@ -43,6 +43,27 @@
...
+
+      describe 'for webp format' do
+        before do
+          SiteSetting.authorized_extensions = 'webp|png'
+        end
+
+        let(:filename) { "webp_as.png" }
+        let(:file) { file_from_fixtures(filename) }
+
+        it 'should not correct the coerce filename' do
```
Comment (tgxworld) [inline spec/lib/upload_creator_spec.rb]: @SamSaffron Is that what you meant?

Comment (SamSaffron) [reply spec/lib/upload_creator_spec.rb]: actually the particular case I mean we should test... take a legit `webp` file, rename it to file.bin. Allow `bin` uploads, upload the `bin` ... it should still be a `.bin`.

This is not the case for png/jpeg cause we would correct the filename.

**Comment by tgxworld** [comment]
Build failure will have to be fixed by the plugin

File: spec/lib/upload_creator_spec.rb [hunk 2]
```diff
@@ -43,6 +43,27 @@
...
+
+      describe 'for webp format' do
+        before do
+          SiteSetting.authorized_extensions = '.webp|.bin'
+        end
+
+        let(:filename) { "webp_as.bin" }
+        let(:file) { file_from_fixtures(filename) }
+
+        it 'should not correct the coerce filename' do
```
Comment (SamSaffron) [inline spec/lib/upload_creator_spec.rb]: should not correct the coerce filename => should not coerce filename

:)

Comment (SamSaffron) [reply spec/lib/upload_creator_spec.rb]: 29k is giant for a webp maybe we can make a smaller one for tests we should be able to swing a webp in 1000 bytes

**Comment by SamSaffron** [comment]
this is good to be merged @tgxworld  I would prefer a smaller webp file first though

File: spec/lib/upload_creator_spec.rb (hunk 2 above)
Comment (tgxworld) [reply spec/lib/upload_creator_spec.rb]: found a webp file that is 26 bytes.

**Pull request merged**